libs/sdk-core/src/breez_services.rs

```diff
@@ -842,6 +852,16 @@ impl BreezServices {
         }
     }
 
+    /// Get the static backup data from the peristent storage.
+    /// This data enables the user to recover the node in an external core ligntning node.
+    /// See here for instructions on how to recover using this data: https://docs.corelightning.org/docs/backup-and-recovery#backing-up-using-static-channel-backup
+    pub fn static_backup(request: StaticBackupRequest) -> SdkResult<StaticBackupResponse> {
+        let storage = SqliteStorage::new(request.working_dir);
```

Comment:
I'm a bit late to the conversation but, isn't this `working_dir` almost always the same with [SDK Config's working_dir](https://github.com/breez/breez-sdk/blob/a441154a106eb7c4dd70a49453bd73c98f94edaa/libs/sdk-core/src/models.rs#L468)?

Reply:
Yes, but from what I understand, the SDK config is not available here to provide the `working_dir` because this is a static method, not a service method called on an instance of `BreezServices`.

See here for more background: https://github.com/breez/breez-sdk/pull/448#discussion_r1322070122